src/Documentation/RuleSetDocumentationGenerator.php

```diff
@@ -52,7 +52,7 @@ public function generateRuleSetsDocumentation(RuleSetDescriptionInterface $defin
 
         $doc .= "\n\n";
 
-        $rules = $definition->getRules();
+        $rules = array_filter($definition->getRules());
```

Comment:
This would filter out the rules with empty config (`[]`), although I think we have a test requiring `true` to be used instead of `[]` when passing no configuration.

Reply:
@jrmajor , can you link the test? I was under impression that empty array is OK

Reply:
Indeed, we're not supposed to pass `[]` but I updated the code anyway.

Reply:
@keradus You're correct, I might have been thinking about `RuleSetTest::testThatDefaultConfigIsNotPassed`, but it tests something else.